db/migrate/20220113175132_add_text_to_post.rb

```diff
@@ -0,0 +1,5 @@
+class AddTextToPost < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :text, :string
```

Comment:
**Optional**

- [ ] It will be nice to change the type from `string` to `text` to adhere to the requirement.